man/man8/zpool.8

```diff
@@ -1591,6 +1591,21 @@ Display full paths for vdevs instead of only the last component of the path.  Th
 .sp
 .ne 2
 .na
+\fB\fB-r\fR\fR
+.ad
+.RS 12n
+.rt
```

Comment:
The `.rt` and `.mk` were just dropped from the man pages will upstream OpenZFS commit 930b0d4c77b9dfdfbc48e7789233b758560e7d84, and follow up commit 81b4c075eca2c4c4129933a012cd354f75201f69.  You'll need to remove them.